src/charm.py

```diff
@@ -40,11 +37,22 @@
 logger = logging.getLogger(__name__)
 
 
-class ZooKeeperCharm(CharmBase):
+class ZooKeeperCharm(CharmWithRelationData):
     """Charmed Operator for ZooKeeper."""
 
     def __init__(self, *args):
-        super().__init__(*args)
+        super().__init__(
+            peer_app_secrets=["sync-password", "super-password"],
+            peer_unit_secrets=[
+                "ca-cert",
+                "csr",
+                "certificate",
+                "truststore-password",
+                "keystore-password",
+                "private-key",
+            ],
```

Comment:
**todo:** Could we move these to `src/literals.py`?

Can be two lists, `PEER_APP_SECRETS` and `PEER_UNIT_SECRETS`

Reply:
Or maybe as class constants within `ZKCluster` and `ZKServer` respectively?
```python
class ZKCluster:
    SECRETS = ["sync-password", "super-password"]
```